server/src/test/java/org/elasticsearch/snapshots/SnapshotResiliencyTests.java

```diff
@@ -594,11 +582,16 @@ public void testBulkSnapshotDeleteWithAbort() {
                 createIndexResponse -> client().admin().cluster().prepareCreateSnapshot(repoName, snapshotName)
                         .setWaitForCompletion(true).execute(createSnapshotResponseStepListener));
 
-        final StepListener<CreateSnapshotResponse> createOtherSnapshotResponseStepListener = new StepListener<>();
+        final int inProgressSnapshots = randomIntBetween(1, 5);
```

Comment:
Here it was easy to expand coverage but unfortunately `SnapshotResiliencyTests` don't really cover repository exception handling so that's where `ConcurrentSnapshotITs` come from.

Reply:
`IOExceptions` thrown by the repository. We don't have any simulation of that in `SnapshotResiliencyTests`